src/ChannelServer/Database/Account.cs

```diff
@@ -87,6 +88,26 @@ public void AddChatMacro(ChatMacro macro)
 		/// <returns></returns>
 		public ChatMacro[] GetChatMacros()
 		{
+			// Initialize with the default set of macros if none exist.
+			if (_chatMacros.Count() == 0)
+			{
+				var macroData = ChannelServer.Instance.Data.ChatMacroDb
+					.Where(x => x.Id <= 10) // maximum number of chat macros allowed.
```

Comment:
Should all 10 be added/sent? I get only the 7 defaults that actually have a pose id sent to me on iToS.